sites/svelte.dev/src/routes/_components/Example.svelte

```diff
@@ -11,7 +11,7 @@
 
 	$: {
 		if (repl) {
-			fetch(`/examples/api/${id}.json`)
+			fetch(`https://svelte.dev/playground/api/${id}.json`)
```

Comment:
Do these endpoints allow CORS?

This also seems like it would make the main site's examples appear here in the v4 site, which we presumably don't want to do, because they'd have runes syntax. I think we need to be doing something else here.

Reply:
just remove the examples section then 😄 it's not like this landing page needs to sell anything anymore

Reply:
found a nice easy solution dc321b21b115c5a63db705a8976012b728fc7702